Code-review/index.html

```diff
@@ -0,0 +1,197 @@
+                    <br>
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
+            </section>
+            <img  class="myimg" src="https://th.bing.com/th/id/OIP.Jciq-y05-b4-VwDB_iUpQwHaEH?w=300&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7" alt="image1"/>
+            <section class="section1">
+                <div class="info">
+                    <span style="font-weight: bold;">
+                        Contributing to open source part II
+                    </span>
+                    <br>
+                    
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
+            </section>
+            <img  class="myimg" src="https://th.bing.com/th/id/OIP.Jciq-y05-b4-VwDB_iUpQwHaEH?w=300&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7" alt="image1"/>
+            <section class="section1">
+                <div class="info">
+                    <span style="font-weight: bold;">
+                        Contributing to open source part II
+                    </span>
+                    <br>
+                    
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
+            </section>
+            <img  class="myimg" src="https://th.bing.com/th/id/OIP.Jciq-y05-b4-VwDB_iUpQwHaEH?w=300&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7" alt="image1"/>
+            <section class="section1">
+                <div class="info">
+                    <span style="font-weight: bold;">
+                        Contributing to open source part II
+                    </span>
+                    <br>
+                    
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
+            </section>
+            <img  class="myimg" src="https://th.bing.com/th/id/OIP.Jciq-y05-b4-VwDB_iUpQwHaEH?w=300&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7" alt="image1"/>
+            <section class="section1">
+                <div class="info">
+                    <span style="font-weight: bold;">
+                        Contributing to open source part II
+                    </span>
+                    <br>
+                    
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
+            </section>
+            
+            
+</div>
+<div>
+    <h1>Youtube video page</h1>
+</div>
```

Comment:
It's rare to have a heading or the title at the bottom-most part of the page. I would suggest your put it in the paragraph and give it the required styling. What do you think?🤔

Reply:
- [ ] You have done great work pointing this out to the student. However `it is rare` is not a good reason and is ambiguous. You could say, this should be a paragraph. Headings usually have a block of text under them but this one does not. That's why I suggest that you make it a paragraph. sounds better, not so?  😄